Library/Formula/apkpatch.rb

```diff
@@ -0,0 +1,21 @@
+class Apkpatch < Formula
+  desc "Tool for build/merge Android Patch file(.apatch)"
+  homepage "https://github.com/alibaba/AndFix"
+  url "https://raw.githubusercontent.com/alibaba/AndFix/master/tools/apkpatch-1.0.3.zip"
+  sha256 "0f94cd2a675af89ff3990f4b6490dd70877ad4eb109f0cdbefece1f9a73e4512"
+
+  def install
+    rm_rf Dir["*.bat"]
+
+    chmod 0755, Dir["*"]
+    libexec.install Dir["*"]
+
+    Dir.glob("#{libexec}/*.sh") do |script|
+      bin.install_symlink script => File.basename(script, ".sh")
+    end
+  end
+
+  test do
+    system "#{bin}/apkpatch", "help"
```

Comment:
Can’t we write a test that does something more substantial than e.g. `help`/`version`? Could we e.g. create a dummy `.apatch` file and apply it on a sample Android app?